src/toolkits/object_detection/object_detector.cpp

```diff
@@ -1347,5 +1354,31 @@ void object_detector::update_model_metrics(gl_sframe data,
   add_or_update_state(metrics);
 }
 
+void object_detector::change_annotations_type(
+    gl_sframe& data, const std::string& annotations_name) {
+  auto change_annotations_type = [=](const flexible_type& annotation) {
```

Comment:
Why use a capture list of `[=]` instead of the simpler `[]`?

Reply:
🧐 Somehow in the previous code I used a outside local value and forget to delete it after I changed my implementation.
Thanks!